nats/msgschema.go

```diff
@@ -153,6 +155,17 @@ func NewResultMsg(startedAt time.Time, result interface{}, err error) ResultMsg
 	return resultMsg
 }
 
+func EventFilter(accountId string) string {
+	tokens := []string{
+		accountId,
+		"notify",
+		"*",
+		"event",
```

Comment:
Create new const at top of file (similar to `HopsMessageId`, e.g. `SourceMessageId`)